src/realm.h

```diff
@@ -3039,7 +3039,6 @@ typedef enum realm_sync_errno_session {
     RLM_SYNC_ERR_SESSION_DIVERGING_HISTORIES = 211,
     RLM_SYNC_ERR_SESSION_BAD_CHANGESET = 212,
     RLM_SYNC_ERR_SESSION_SUPERSEDED = 213,
```

Comment:
SUPERSEDED is never going to be sent by the server. Let's nuke it from orbit too.